src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/Shared/ConfigHandler.js

```diff
@@ -0,0 +1,119 @@
+class ConfigLoader {
```

Comment:
I think the only two small things I have in this file are:
* I seem to remember you changed CfgParse to IniParser, so I just want to make sure I'm looking at the right stuff
* The methods and classes all need javadoc.

Reply:
I changed loadCfg to loadIni to more accurately represent the particular file format it's meant to parse.  The class loads both ini and XML files (both "normal" XML and the warped kind Asobo are using with multiple roots) so "Config" felt, to me at least, like it remained more appropriate there.